src/collection.ts

```diff
@@ -457,4 +482,26 @@ export class Collection<T extends TigrisCollectionType> extends ReadOnlyCollecti
 			});
 		});
 	}
+
+	/**
+	 * Updates a single document in collection
```

Comment:
please put a notice on what the behavior is if filter matches more than one doc.